ffi/diplomat/src/fixed_decimal.rs

```diff
@@ -19,13 +21,67 @@ pub mod ffi {
         pub success: bool,
     }
 
+    /// How to round digits when constructing an ICU4XFixedDecimal from a
+    /// floating point number
+    pub enum ICU4XFixedDecimalRoundingMode {
+        /// Truncate leftover digits
+        Truncate,
+        ///  Round up from 0.5
+        HalfExpand,
+    }
+
     impl ICU4XFixedDecimal {
         /// Construct an [`ICU4XFixedDecimal`] from an integer.
+        ///
         /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html) for more information.
         pub fn create(v: i32) -> Box<ICU4XFixedDecimal> {
             Box::new(ICU4XFixedDecimal(FixedDecimal::from(v)))
         }
 
+        /// Construct an [`ICU4XFixedDecimal`] from an float, with enough digits to recover
+        /// the original floating point in IEEE 754 without needing trailing zeros
+        ///
+        /// See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html#method.from_f64) for more information.
+        pub fn create_from_f64(f: f64) -> Option<Box<ICU4XFixedDecimal>> {
```

Comment:
Nit: create_from_f64_with_max_precision

You know how I feel about making this have the appearance of being the default function. ;)